.github/workflows/docker-publish.yml

```diff
@@ -106,5 +106,6 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+          outputs: type=image,name=target,annotation-index.org.opencontainers.image.description=The NordVPN client for Docker
```

Comment:
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#adding-a-description-to-multi-arch-images

Reply:
OK, this in place before change to a template approach.